integration/src/integration/java/pl/allegro/tech/hermes/integration/PublishingTest.java

```diff
@@ -67,6 +67,21 @@ public void shouldPublishAndConsumeMessage() {
     }
 
     @Test
+    public void shouldReturn429ForQuotaViolation() {
+        // given
+        Topic topic = operations.buildTopic("publishAndConsumeGroup", "topic");
+        TestMessage message = TestMessage.of("content", StringUtils.repeat("X", 60_000));
```

Comment:
this `60_000` - where does it come from? if this is based on config, please add this as test config instead of relying on defaults and add comment that describes origin of this magic number